portality/view/api_v1.py

```diff
@@ -40,61 +40,322 @@ def list_operations():
+                          type: object
+                          title: license
+                          properties:
+                            title:
+                              type: string
+                            url:
+                              type: string
+                            NC:
+                              type: boolean
+                            ND:
+                              type: boolean
+                            embedded_example_url:
+                              type: string
+                            SA:
+                              type: boolean
+                            type:
+                              type: string
+                            BY:
+                              type: boolean
+                      title:
+                        type: string
+                      author_pays_url:
+                        type: string
+                      country:
+                        type: string
+                      link:
+                        type: array
+                        items:
+                          type: object
+                          title: link
+                          properties:
+                            url:
+                              type: string
+                            type:
+                              type: string
+                      provider:
+                        type: string
+                      keywords:
+                        type: array
+                        items:
+                            type: string
+                      oa_start:
+                        type: object
+                      identifier:
+                        type: array
+                        items:
+                          type: object
+                          title: identifier
+                          properties:
+                            type:
+                              type: string
+                            id:
+                              type: string
+                      subject:
+                        type: array
+                        items:
+                          type: object
+                          title: subject
+                          properties:
+                            code:
+                              type: string
+                            terms:
+                              type: string
+                            scheme:
+                              type: string
+                  created_date:
+                    type: string
+                    format: dateTime
+            query:
+              type: string
+            total:
+              type: integer
+            page:
+              type: integer
+      400:
+        description: Bad Request
+    """
+    # get the values for the 2 other bits of search info: the page number and the page size
+    page = request.values.get("page", 1)
+    psize = request.values.get("pageSize", 10)
```

Comment:
hm, @Nimphal pageSize seems to be 10? So definitely go for definiing DEFAULT_PAGE_SIZE in the api common.py file or at least top of this module.